package.json

```
@@ -1082,7 +1082,7 @@
     "jest-styled-components": "7.0.3",
     "jsdom": "^16.4.0",
     "json-schema-typed": "^8.0.1",
-    "json5": "^1.0.1",
+    "json5": "2.2.2",
```

Any particular reason why we're not leaving the minor version range open with a `^`?

```suggestion
    "json5": "^2.2.2",
```

---

Thanks! I missed this when the file was automatically updated.